R/annotation-data.R

```diff
@@ -17,71 +17,44 @@
 #'
 #' @examples
 #' 
-#' txdbPath <- list.files(system.file('extdata/testdata/promoterAnnotation', 
-#'                                    package = 'proActiv'), full.names = TRUE, pattern = 'sqlite')
+#' txdbPath <- system.file('extdata/vignette/annotations/',
+#'                             'gencode.v34.annotation.chr22.sqlite', 
+#'                              package = 'proActiv')
 #' txdb <- AnnotationDbi::loadDb(txdbPath)
 #' promoterAnnotation <- preparePromoterAnnotation(txdb = txdb,
 #'                                                 species = 'Homo_sapiens')
 #' 
```

Comment:
To shorten `preparePromoterAnnotation` function length, shift file parsing to a helper function `parseTxDb` found in `annotation-data-helper.R`